packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -117,27 +123,27 @@ const translation = {
         "Attestation": "J'ai lu, je comprends et j'accepte les conditions."
     },
     "Tour": {
-        "GuideTitle": "Welcome to Aries Bifold (FR)",
-        "WouldYouLike": "Would you like some guidance on how to use Aries Bifold? (FR)",
-        "UseAppGuides": "Use app guides (FR)",
-        "DoNotUseAppGuides": "Don't use app guides (FR)",
-        "AddAndShare": "Add and share credentials (FR)",
-        "AddAndShareDescription": "To add and use credentials you scan a QR code displayed by the service provider. (FR)",
-        "Notifications": "Notifications (FR)",
-        "NotificationsDescription": "After you scan a QR code, the credential offer or proof request will appear here, as well as other notable events. (FR)",
-        "YourCredentials": "Your credentials (FR)",
-        "YourCredentialsDescription": "Your added digital credentials appear here. You can review credential details and add and delete credentials. (FR)",
-        "Skip": "Skip (FR)",
-        "Next": "Next (FR)",
-        "Back": "Back (FR)",
-        "Done": "Done (FR)",
+        "GuideTitle": "Bienvenue à Aries Bifold",
+        "WouldYouLike": "Souhaitez-vous des conseils sur la façon d'utiliser Aries Bifold?",
+        "UseAppGuides": "Utiliser les guides d'applications",
+        "DoNotUseAppGuides": "N'utilisez pas les guides d'application",
+        "AddAndShare": "Ajouter et partager des justificatifs d'identité",
+        "AddAndShareDescription": "Pour ajouter et utiliser des justificatifs d'identité, vous scannez un code QR affiché par le fournisseur de services.",
+        "Notifications": "Avis",
+        "NotificationsDescription": "Après avoir scanné un code QR, l'offre de justificatif d'identité ou la demande de preuve apparaîtra ici, ainsi que d'autres événements notables.",
+        "YourCredentials": "Vos justificatifs d'identité",
+        "YourCredentialsDescription": "Vos justificatifs d'identité numériques ajoutées apparaissent ici. Vous pouvez consulter les justificatifs d'identité et ajouter et supprimer des justificatifs d'identité.",
+        "Skip": "Passer",
+        "Next": "Suivant",
+        "Back": "Précédent",
+        "Done": "Terminer",
       },
     "CameraDisclosure": {
-        "AllowCameraUse": "Allow camera use (FR)",
+        "AllowCameraUse": "Autoriser l'utilisation de l'appareil photo",
         "CameraDisclosure": "La caméra est utilisée pour scanner les codes QR pour un traitement immédiat sur l'appareil. Aucune information sur les images n'est stockée, utilisée à des fins d'analyse ou partagée.",
-        "ToContinueUsing": "To continue using the Aries Bifold scan feature, please allow camera permissions. (FR)",
-        "Allow": "Allow (FR)",
-        "OpenSettings": "Open settings (FR)",
+        "ToContinueUsing": "Pour continuer à utiliser la fonction de balayage du Portefeuille QC, veuillez autoriser les autorisations de caméra.",
```

Comment:
Is this a better way to say this in French and to keep consistent the use of "caméra" instead of "l'appareil photo":  "Pour continuer à utiliser la fonction de balayage du Portefeuille QC, autoriser l'utilisation de caméra."